docs/io/fits/usage/cloud.rst

```diff
@@ -0,0 +1,268 @@
+    missing. See :ref:`installing-astropy` for details on installing optional
+    dependencies.
+
+Working with Amazon S3 access credentials
+-----------------------------------------
+
+Note that we used the ``fsspec_kwargs`` parameter in the example above to pass
+extra arguments to the `fsspec.open` function.  Specifically, we passed the
+``anon=True`` parameter to indicate that we want to retrieve data in an
+anonymous way without providing Amazon cloud access credentials.
+For convenience, Astropy will pass ``anon=True`` by default if a path starts
+with ``s3://`` and ``fsspec_kwargs`` is unspecified.
+
+In some cases you may want to access data stored in an Amazon S3 data bucket
+that is private or uses the "Requester Pays" feature. You will have to provide
+a secret access key in this case. You can use the ``fsspec_kwargs`` parameter
+to provide your key as follows:
+
+.. doctest-skip::
+
+    >>> fsspec_kwargs = {"key": "YOUR-SECRET-KEY-ID",
+    ...                  "secret": "YOUR-SECRET-KEY"}
+    >>> with fits.open(s3_uri, use_fsspec=True, fsspec_kwargs=fsspec_kwargs) as hdul:
+    ...     cutout = hdul[2].section[10:20, 30:50]
+
+.. warning::
+
+    Including secret access keys inside Python code is dangerous because you
+    may accidentally end up revealing your keys when you share your code with
+    others. A better practice is to store your access keys via a configuration
+    file or environment variables. See the ``s3fs`` documentation for guidance.
+
+
+Using :class:`~astropy.nddata.Cutout2D` with cloud-hosted FITS files
+====================================================================
+
+The examples above used the `ImageHDU.section` feature to download
+small cutouts given a set of pixel coordinates. For astronomical images it is
+often more convenient to obtain cutouts based on a sky position and angular
+size rather than array coordinates. For this reason, Astropy provides the
+`astropy.nddata.Cutout2D` tool which makes it easy to obtain cutouts informed
+by an image's World Coordinate System (`~astropy.wcs.WCS`).
+
+This cutout tool can be used in combination with ``fsspec`` and ``.section``.
+For example, assume you happen to know that the image we opened above contains
+a nice edge-on galaxy at the following position::
+
+    >>> # Approximate location of an edge-on galaxy
+    >>> from astropy.coordinates import SkyCoord
+    >>> position = SkyCoord('10h01m41.13s 02d25m20.58s')
+
+We also know that the radius of the galaxy is approximately 5 arcseconds::
+
+    >>> # Approximate size of the galaxy
+    >>> from astropy import units as u
+    >>> size = 5*u.arcsec
+
+Given this sky position and radius, we can use `~astropy.nddata.Cutout2D`
+in combination with ``use_fsspec=True`` and ``.section`` as follows:
+
+.. doctest-requires:: fsspec
+
+    >>> from astropy.nddata import Cutout2D
+    >>> from astropy.wcs import WCS
+    ...
+    >>> with fits.open(s3_uri, use_fsspec=True) as hdul:  # doctest: +REMOTE_DATA
```

Comment:
With the `s3_url`, `use_fsspec` is not necessary, right?

Reply:
Agreed.